src/Entity/Donation.php

```diff
@@ -43,58 +53,59 @@ class Donation implements GeoPointInterface
     private $emailAddress;
 
     /**
+     * @var PhoneNumber|null
+     *
      * @ORM\Column(type="phone_number", nullable=true)
      */
     private $phone;
 
     /**
-     * @ORM\Column(length=100, nullable=true)
-     */
-    private $payboxResultCode;
-
-    /**
-     * @ORM\Column(length=100, nullable=true)
+     * @var string|null
+     *
+     * @ORM\Column(length=50, nullable=true)
      */
-    private $payboxAuthorizationCode;
+    private $clientIp;
```

Comment:
This client ip should be moved to transaction no ? I don't really understand its purpose with the new entity `Transaction`

Reply:
Client IP is set only one time when the order(donation) is created after that all transactions received are a responses from paybox to say if it's ok or not and why. No ?

Reply:
Ok got it now. Thanks ;-) 
Could you add a comment about this please?